L1Trigger/CSCTriggerPrimitives/interface/CSCCathodeLCTProcessor.h

```diff
@@ -203,6 +206,12 @@ class CSCCathodeLCTProcessor : public CSCBaseboard {
   std::vector<int> thePreTriggerBXs;
   std::vector<CSCCLCTPreTriggerDigi> thePreTriggerDigis;
 
+  /* data members for high multiplicity triggers */
+  void encodeHighMultiplicityBits(
+      const std::vector<int> halfstrip[CSCConstants::NUM_LAYERS][CSCConstants::NUM_HALF_STRIPS_7CFEBS]);
```

Comment:
@dildick , it is possible to use "const std::vector<int>& ..." ?

Reply:
@civanch yes, I will fix it.